third_party/WebKit/Source/modules/vr/NavigatorVR.h

```diff
@@ -43,7 +43,7 @@ class MODULES_EXPORT NavigatorVR final
   Document* document();
 
   // Queues up event to be fired soon.
-  void enqueueVREvent(VRDisplayEvent*);
+  void enqueueVREvent(Event*);
```

Comment:
Is this still needed? I think we are not using enqueueVREvent anymore. Better leave it as it was.

Reply:
Good point. Will change back.